fhir-persistence-schema/docs/DB2MultiTenancy.md

```diff
@@ -1,16 +1,225 @@
-# The IBM FHIR Server - Db2 Multi-Tenancy
+# The IBM FHIR Server - Multi-Tenancy on Db2
 
-This document outlines the schema design and implementation for the IBM FHIR Server's Db2 Multi-tenancy feature. 
+This document outlines the schema design and implementation for the IBM FHIR Server's Multi-tenancy feature on Db2.
```

Comment:
Where is tenant_query_rewrite.png used?

Reply:
Not referenced at this time, but will be useful for helping explain the Db2 row permission feature more clearly.

Reply:
I'm not sure how I feel about something unreferenced. Maybe just add it inline>?